cita-chain/core/src/libchain/chain.rs

```diff
@@ -1124,7 +1124,7 @@ impl Chain {
                         receipts.len(),
                         hashes.len()
                     );
-                    assert!(false);
+                    panic!();
```

Comment:
Add a message in `panic!(...)` or just use `unreadable!()`.

Reply:
ok